business/tests/Services/PacsMirrorServiceUnitTests.cs

```diff
@@ -0,0 +1,159 @@
+using NUnit.Framework;
+using MetadataDatabase.Controllers;
+using Moq;
+using MetadataDatabase.Services;
+using System.Collections.Generic;
+using MetadataDatabase.Data;
+
+namespace MetadataDatabase
+{
+    [TestFixture]
+    public class PacsMirrorServiceUnitTests
+    {
+        [Test]
+        public void TestMirrorPacsMethodWithNoChanges()
+        {
+            // Arrange
+            var mockSeriesServices = new Mock<ISeriesServices>();
+            mockSeriesServices.Setup(mock => mock.GetAll())
+                .Returns(GetTestDatabaseSeries(4));
+
+            var mockPacsService = new Mock<IPacsService>();
+            mockPacsService.Setup(repo => repo.GetSeriesAsync())
+                .ReturnsAsync(GetTestPacsSeries(4));
+
+            IPacsMirrorService pacsMirrorService = new PacsMirrorService(
+                mockSeriesServices.Object, 
+                mockPacsService.Object);
+            // Act
+            pacsMirrorService.MirrorPacs();
+            // Assert
+            Assert.AreEqual(true, true);
+            mockSeriesServices.Verify(mock => mock.Delete(It.IsAny<string>()), Times.Never());
+            mockSeriesServices.Verify(mock => mock.Create(It.IsAny<SeriesDto>()), Times.Never());
+        }
+
+        [Test]
+        public void TestMirrorPacsMethodWith2SeriesToDelete()
+        {
+            // Arrange
+            var mockSeriesServices = new Mock<ISeriesServices>();
+            mockSeriesServices.Setup(mock => mock.GetAll())
+                .Returns(GetTestDatabaseSeries(6));
+
+            var mockPacsService = new Mock<IPacsService>();
+            mockPacsService.Setup(repo => repo.GetSeriesAsync())
+                .ReturnsAsync(GetTestPacsSeries(4));
+
+            IPacsMirrorService pacsMirrorService = new PacsMirrorService(
+                mockSeriesServices.Object,
+                mockPacsService.Object);
+            // Act
+            pacsMirrorService.MirrorPacs();
+            // Assert
+            Assert.AreEqual(true, true);
```

Comment:
Idem, I don't understand the use of this assert.

Reply:
To delete ;-)